lib/ansible/modules/system/selinux_permissive.py

```diff
@@ -19,35 +19,36 @@
 module: selinux_permissive
 short_description: Change permissive domain in SELinux policy
 description:
-  - Add and remove domain from the list of permissive domain.
+  - Add and remove a domain from the list of permissive domains.
 version_added: "2.0"
 options:
   domain:
     description:
-        - "the domain that will be added or removed from the list of permissive domains"
+        - The domain that will be added or removed from the list of permissive domains.
     required: true
   permissive:
     description:
-        - "indicate if the domain should or should not be set as permissive"
+        - Indicate if the domain should or should not be set as permissive.
     required: true
     type: bool
   no_reload:
     description:
-        - "automatically reload the policy after a change"
-        - "default is set to 'false' as that's what most people would want after changing one domain"
-        - "Note that this doesn't work on older version of the library (example EL 6), the module will silently ignore it in this case"
+        - Automatically reload the policy after a change.
```

Comment:
@major I find this confusing, and I'm hoping you can help clarify the behavior. The parameter is called `no_reload` (instead of `reload`), and the double negative seems like it should behave thus:
- if `no_reload` is set to `no`, the module reloads
- if `no_reload` is set to `yes`, the module does not reload
But the line below this suggests that when `no_reload` is set to `no`, the module does NOT reload. I know you're not changing the parameter in this PR, but can you clarify the behavior?